presidio-analyzer/Dockerfile

```diff
@@ -2,16 +2,32 @@ FROM python:3.9-slim
 
 ARG NAME
 ARG NLP_CONF_FILE=presidio_analyzer/conf/default.yaml
+ARG ANALYZER_CONF_FILE=presidio_analyzer/conf/default_analyzer.yaml
+ARG RECOGNIZER_REGISTRY_CONF_FILE=presidio_analyzer/conf/default_recognizers.yaml
 ENV PIPENV_VENV_IN_PROJECT=1
 ENV PIP_NO_CACHE_DIR=1
+
+ENV ANALYZER_CONF_FILE=${ANALYZER_CONF_FILE}
+ENV RECOGNIZER_REGISTRY_CONF_FILE=${RECOGNIZER_REGISTRY_CONF_FILE}
+ENV NLP_CONF_FILE=${NLP_CONF_FILE}
```

Comment:
I think that the ENV lines are redundant

Reply:
The app.py reads those values from env variables. Is there a better option to have the user introduce those parameters and have them set as env variables? Because these are paths, I wasn't sure if passing env variables to the docker build call would end up having the right path, but I have quite limited experience with Docker.

Reply:
Got it, makes sense then